app/controllers/admin/statistics_announcement_tags_controller.rb

```diff
@@ -39,4 +42,15 @@ def selected_taxons
   def invisible_taxons
     params["taxonomy_tag_form"].fetch("invisible_taxons", "").split(",")
   end
+
+  def get_layout
+    design_system_actions = []
+    design_system_actions += %w[edit] if preview_design_system?(next_release: false)
+
+    if design_system_actions.include?(action_name)
```

Comment:
There's only one action here so i think we can just do 
```
def get_layout
    if preview_design_system?(next_release: false)
      "design_system"
    else
      "admin"
    end
  end
```

Reply:
Agree, yes, can refactor this down.